### DISCUSSION TITLE
[@types/prompts] 2.4.0 should be unpublished/superseded

### DISCUSSION THREAD

**Discussion by vlindhol** [DefinitelyTyped/DefinitelyTyped#59868] [Issues with a @types package]
`@types/prompts` is at version 2.0.14 which e.g. has a typo fix for an attribute that was fixed long ago (`seperator` -> `separator`). However, [looking at the NPM versions page](https://www.npmjs.com/package/@types/prompts/v/2.0.14?activeTab=versions), there is also a weird 2.4.0 package published 3 years ago that takes precedence when installing the package.

Seems like a versioning mistake, and `@types/prompts@2.4.0` should be unpublished from NPM, or a new release cut as e.g. 2.4.1 with up-to-date code. I don't know if this is something I can do myself, sorry if it is!

[a comment by an automated account is not shown]

**Comment by kamontat** [comment]
I totally agree with you that v2.4.0 should be unpublished. However, I don't have permission to do so. We need someone from DT teams to unpublish them.

This happen because wrong parsing version on [this](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35147) PR
ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35147

**Reply by vlindhol** [reply]
Thanks! Created issue https://github.com/DefinitelyTyped/DefinitelyTyped/issues/59890 which will hopefully reach someone from the DT team.
